lib/link_thumbnailer/model.rb

```diff
@@ -9,7 +9,7 @@ def to_json(*args)
 
     def sanitize(str)
       return unless str
-      str.strip.gsub(/[\r\n\f]+/, "\n")
+      str.encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '').strip.gsub(/[\r\n\f]+/, "\n")

```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.